modules/sdk-coin-btc/test/unit/util/utxoCoins.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @prettier
+ */
+import * as utxolib from '@bitgo/utxo-lib';
+import { AbstractUtxoCoin } from '@bitgo/abstract-utxo';
+import { TestBitGo } from '@bitgo/sdk-test';
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { Btc, Tbtc } from '../../../src';
+
+export const defaultBitGo = TestBitGo.decorate(BitGoAPI, { env: 'mock' });
+
+function getUtxoCoins(): AbstractUtxoCoin[] {
+  defaultBitGo.safeRegister('btc', Btc.createInstance);
+  defaultBitGo.safeRegister('tbtc', Tbtc.createInstance);
+  return [new Btc(defaultBitGo), new Tbtc(defaultBitGo)];
+}
```

Comment:
this significantly reduces test coverage

Reply:
I did not mean to remove the original tests from bitgo, will revert & exclude btc/tbtc from the tests that run in that package.

Reply:
the test suites in this directory are supposed to run over *all* utxo coins though

Reply:
the original tests are still in this branch. All of the utxo unit tests are unchanged:

https://github.com/BitGo/BitGoJS/tree/aloe/btcRefactor/modules/bitgo/test/v2/unit/coins/utxo

the code you're seeing in this file is the same content as what's in `modules/bitgo/test/v2/unit/coins/utxo` , except it only runs the tests for `Btc` and `Tbtc`, since those are the only coins in this module. Eventually when each coin is in its own package we will be able to completely eliminate the utxo tests in bitgo. If anything, we should exclude `Btc` and `Tbtc` from the tests run there, since they are now being duplicated.

Reply:
as discussed on slack: we want the test suite to live in once place only